Framework/Geometry/src/Rasterize.cpp

```diff
@@ -122,6 +122,25 @@ calculateHollowCylinder(const V3D &beamDirection,
                                  numAnnuli);
 }
 
+const V3D calculatePos(const double phi, const double R, const double z,
+                       const std::array<V3D, 3> coords) {
+  const auto x_prime = coords[0];
```

Comment:
I think these should be `const auto &` to help the compiler in knowing that you just want friendlier names rather than copies of the data.